### PR TITLE
freebsd 11.1 g++7 build

### PR DESCRIPTION
**Pull request by lyrra**
build for freebsd 11.1 g++7 (really moving to g++ std11)

### CODE REVIEW

File: src/bitcoin-tx.cpp [hunk 1]
```diff
@@ -250,7 +248,7 @@ static void MutateTxAddInput(CMutableTransaction& tx, const std::string& strInpu
     // extract the optional sequence number
     uint32_t nSequenceIn=std::numeric_limits<unsigned int>::max();
     if (vStrInputParts.size() > 2)
-        nSequenceIn = std::stoul(vStrInputParts[2]);
+        nSequenceIn = stoul(vStrInputParts[2], 0, 10);
```
Comment (laanwj) [inline src/bitcoin-tx.cpp]: Please keep using the fully-qualified name

**Comment by laanwj** [comment]
Did anything change in the C++ standard to require explicitly passing these arguments instead of implicitly by default?

File: src/bitcoin-tx.cpp [hunk 2]
```diff
@@ -22,8 +22,6 @@
 #include "utilmoneystr.h"
 #include "utilstrencodings.h"
 
-#include <stdio.h>
-
```
Comment (paveljanik) [inline src/bitcoin-tx.cpp]: I think that this is the only valid change right now.

**Comment by lyrra** [comment]
Please wait, I'll gmake clean && gmake without the patch :)

**Comment by lyrra** [comment]
Please disregard my patch! I have now compiled&installed&runned bitcoind with both Clang and g++7.
My patch sprung from trying to fix build on an older freebsd with older compiler and boost (pre-c++11).

**Comment by fanquake** [comment]
Thanks @lyrra, I'll close this PR then.

**Pull request closed without merging**